src/applications/applications.service.ts

```
@@ -218,4 +218,83 @@ export class ApplicationsService {
       data: updatedApplication,
     };
   }
+
+  async exportApplicationsCsv(benefitId: string, reportType: string): Promise<string> {
```

ESLint

[error] Replace `benefitId:·string,·reportType:·string` with `⏎····benefitId:·string,⏎····reportType:·string,⏎··`

(prettier/prettier)